mtvsrs/urls.py

```diff
@@ -16,11 +16,19 @@
 """
 
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 from . import views
+from django.contrib.auth import views as auth_views
+
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", views.home_page, name="home"),
     path("show/<int:show_id>/", views.show_page, name='show'),
+    path("register", views.register_user, name="register"),
+    path("", include("django.contrib.auth.urls")),
```

Comment:
This will never be reached, in line 26 going to root url will always match home_page first. See if this is necessary